cmd/ci-operator/main.go

```diff
@@ -280,6 +282,9 @@ func bindOptions(flag *flag.FlagSet) *options {
 	flag.StringVar(&opt.branch, "branch", "", "Branch of the project (used by configresolver)")
 	flag.StringVar(&opt.variant, "variant", "", "Variant of the project's ci-operator config (used by configresolver)")
 
+	// flag to disable configresolver for rehearse
+	flag.BoolVar(&opt.rehearse, "rehearse", false, "Run in rehearse mode (disable configresolver calls)")
```

Comment:
I am not a huge fan of test logic leaking into production code. Couldn't we just remove the org/repo/branch/URL flags when doing a rehearsal?

Reply:
That was one of the other ideas I had about how to handle this. I thought that doing it this way would be a bit cleaner, but either way works.

Reply:
Changed it so now it removes the 5 flags the configure the configresolver instead.